services/security_group_manager/src/main/java/com/futurewei/alcor/securitygroup/controller/SecurityGroupRuleController.java

```diff
@@ -15,24 +15,28 @@
 */
 package com.futurewei.alcor.securitygroup.controller;
 
+import com.futurewei.alcor.common.stats.DurationStatistics;
 import com.futurewei.alcor.securitygroup.service.SecurityGroupRuleService;
 import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupRule;
 import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupRuleBulkJson;
 import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupRuleJson;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 import java.util.List;
 import static com.futurewei.alcor.securitygroup.utils.RestParameterValidator.*;
 
 @RestController
+@ComponentScan(value = "com.futurewei.alcor.common.stats")
```

Comment:
@chenpiaoping Can you test the SG APIs from API GW to see if anything is expected? Recently there is a minor change of List SG API introduced by PR #https://github.com/futurewei-cloud/alcor/pull/304. The List SG contract was not fully compatible wit Neutron. Can you check others as well. 

Thanks.

Reply:
You can do it a separate/small PR if any change is needed.

Reply:
ok, i will do that.